gitlab/unit-tests.sh

```diff
@@ -20,7 +20,7 @@ echo "UPDATE user SET teamid = 1 WHERE userid = 1;" | mysql domjudge_test
 cp webapp/.env.test /opt/domjudge/domserver/webapp/
 
 # We also need the composer.json for PHPunit to detect the correct directory.
-cp composer.json /opt/domjudge/domserver/
+cp webapp/composer.json /opt/domjudge/domserver/
```

Comment:
Shouldn't this then move to `/opt/domjudge/domserver/webapp`?

Reply:
Yes, not sure why tests did work.
```suggestion
cp webapp/composer.json /opt/domjudge/domserver/webapp/
```